test/parallel/test-inspector-workers-flat-list.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+
+const { Session } = require('inspector');
+const { Worker, isMainThread, parentPort } = require('worker_threads');
+
+const MAX_DEPTH = 3;
+
+let rootWorker = null;
+
+function runTest() {
```

Comment:
`const runTest = common.mustCall(() => { ...`, to make sure that this is actually run?

Reply:
Done